client/src/app/contexts/AuthContext.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import React from "react";
+import { Auth0Provider } from "@auth0/auth0-react";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+// Props for AuthContext
+interface AuthContextProps {
+  children: React.ReactNode;
+}
+
+// AuthContext to wrap the application around
+const AuthContext: React.FC<AuthContextProps> = ({ children }) => {
+  return (
+    <Auth0Provider
+      domain={process.env.NEXT_PUBLIC_DOMAIN || ""}
+      clientId={process.env.NEXT_PUBLIC_CLIENTID || ""}
```

Comment:
Do these variables come from your `.env` file or are they just nextjs environment variables that we can just use? If its the former, why is it different from `.env.example`?

I'm mostly curious cause when I'm working on CI/CD right now so its important I know all the env variables that we use.